src/redux/fetchApi.js

```diff
@@ -0,0 +1,60 @@
+import { createAsyncThunk, createSlice } from '@reduxjs/toolkit';
+import axios from 'axios';
```

Comment:
- [ ] You are importing `axios` here but you did not include it as a package in the `package.json` file so I get an error when trying to start your app.

  You can fix this by installing `axios`:

  ```
  npm install axios
  ```

  ![image](https://user-images.githubusercontent.com/36542986/214864643-e6885299-c66f-4b80-9f91-57c32aa83e50.png)

Reply:
I reinstalled the axios. I don't seem to know why it wasnot working it was working fine on my side.